getSimulationPA.m

```diff
@@ -28,13 +28,14 @@
             xi, ...
             Model.exp{s}, ... = estruct (in getSigmaPointApp)
             op_SP);
-
+        
         % Store the simulation results
         switch Model.exp{s}.scale
             case 'log'
                 % TBD!
-                tmp = arrayfun(@(x) diag(squeeze(SP.my(x,:,:))), 1:size(SP.Cy,1),'UniformOutput',false);
-                my = [my; exp(SP.my + transpose([tmp{:}])/2)];
+                tmp = arrayfun(@(x) diag(squeeze(SP.Cy(x,:,:))), 1:size(SP.Cy,1),'UniformOutput',false);
```

Comment:
wait... why are we precisely using Cy-related things here? I think we shouldn't, it feels wrong... in the PA-code there should not be anything using Cy, I think...

Reply:
Okay, this part I recomputed. It's correct. The reason is:
Let's call `yf` the final output and `ys` the model simulation. Then, using this log-transformation, we assume that y is approximately normally distributed, with mean `mean_ys` and std `sigma_ys`. The final, transformed variable will then have the expectation `mean_yf = exp(mean_ys + sigma_ys^2 / 2)` and the variance `sigma_yf = exp(2*mean_ys + sigma_ys^2) * (exp(sigma_ys^2) - 1). This `Cy` is just the `sigma_ys` which we see here.

Reply:
ah, yes!!!! it is log-normal distributed!!!! OMG, it took me such a long time....